### PR TITLE
fix: changing docker-compose to docker compose in makefile

### LINKED ISSUE #3654: changing docker-compose to docker compose in makefile
# This is a Feature Request
Hello team
I was setting up jx-docs locally and I run `make compose-up` to start the docker container.
It shows this error

```bash
zsh: command not found: docker-compose
```

I'm using
OS: Ubuntu 20.04
Make Version: 4.2.1
Docker version: 20.10.12, build 20.10.12-0ubuntu2~20.04.1
Docker Compose version: v2.10.2

# What would you like to be added
On installing docker, docker compose is installed not docker-compose.
because `docker-compose` is now deprecated and `docker compose` is the v2.

So I think, it would be nice if we use docker compose for jx-docs.

# Why is this needed
To start the jx-docs container we need to install docker-compose which is a deprecated tools written in python, the v2 is rewritten in golang.

The official documentation talks about it, https://github.com/docker/compose

# Comments
While searching the solution I came across this [link](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose)

### PR DESCRIPTION
**Pull request by rajatgupta24**
# Description
This PR changes the docker-compose to docker compose.

**_Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change._**

This PR does not add any new dependency.

Fixes #3654 

# Checklist:

- [x] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [ ] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [x] Any dependent changes have already been merged.

### CODE REVIEW

**Comment by rajatgupta24** [comment]
I have not installed docker-compose and I tried to run `make compose-up` which runs `docker compose up -d server` and this is the output
image

I have also tried all the other commands as well.

**Comment by ankitm123** [comment]
So the reason this works is because compose v2 was recently announced, which is a go rewrite of docker composed v1 which was in python. 
https://www.docker.com/blog/announcing-compose-v2-general-availability/
So, we need to update the install instructions in the doc contribution guide as well.

**Comment by ankitm123** [comment]
Specifically these docs: https://jenkins-x.io/community/documentation/#docker-compose-method-preferred-approach
Remember to explicitely mention the version of docker cli which has compose v2!

File: content/en/community/documentation/_index.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Use this:
 
 The first thing you'll need to make use of this approach is Docker installed on your local environment.
 How to install a Docker engine depends on your platform etc., so best to head over to [Docker](https://docs.docker.com/install/) to find the right one.
-Next, install docker-compose from the [installation documents](https://docs.docker.com/compose/install/).
+Next, you have install docker compose v2 from the [installation guide](https://docs.docker.com/compose/install/).
```
Comment (rajatgupta24) [inline content/en/community/documentation/_index.md:62]: @ankitm123, is this good?

**Review by ankitm123 — APPROVED** [review]

**Pull request merged**